protoc-gen-openapiv2/internal/genopenapi/template.go

```diff
@@ -2074,3 +2135,13 @@ func getReservedJSONName(fieldName string, messageNameToFieldsToJSONName map[str
 	fieldNames := strings.Split(fieldName, ".")
 	return getReservedJSONName(strings.Join(fieldNames[1:], "."), messageNameToFieldsToJSONName, fieldNameToType)
 }
+
+func find(a []string, x string) int {
+	// This is a linear search but we are dealing with a small number of fields
+	for i, n := range a {
+		if x == n {
+			return i
+		}
+	}
+	return -1
+}
```

Comment:
Nit: newline at the end of the file please.

Reply:
No problem.

Do you format with `go fmt`? If so, might be good to have CI run a check to see if all files are formatted property so it's easier for you the reviewer and for contributors.

Reply:
You're absolutely right, we should do that. I haven't messed around with the CI jobs for a while but I might just set up a new golangci-lint job.

Reply:
Done, you may want to rebase on master in case there are other formatting errors: https://github.com/grpc-ecosystem/grpc-gateway/pull/1817